<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DepositView</class>
 <widget class="QMainWindow" name="DepositView">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>602</width>
    <height>576</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>602</width>
    <height>576</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>602</width>
    <height>576</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="label_19">
    <property name="geometry">
     <rect>
      <x>53</x>
      <y>268</y>
      <width>40</width>
      <height>16</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Налог</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_17">
    <property name="geometry">
     <rect>
      <x>306</x>
      <y>170</y>
      <width>111</width>
      <height>16</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Частичные снятия</string>
    </property>
   </widget>
   <widget class="QSpinBox" name="spinBox_period_of_placement">
    <property name="geometry">
     <rect>
      <x>173</x>
      <y>107</y>
      <width>81</width>
      <height>30</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="minimum">
     <number>1</number>
    </property>
    <property name="maximum">
     <number>1000</number>
    </property>
   </widget>
   <widget class="QRadioButton" name="radioButton_add">
    <property name="geometry">
     <rect>
      <x>34</x>
      <y>22</y>
      <width>181</width>
      <height>16</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Капитализация процентов</string>
    </property>
    <property name="checked">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="label_15">
    <property name="geometry">
     <rect>
      <x>307</x>
      <y>51</y>
      <width>141</width>
      <height>16</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Переодичность выплат</string>
    </property>
   </widget>
   <widget class="Line" name="line_3">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>221</y>
      <width>601</width>
      <height>20</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Orientation::Horizontal</enum>
    </property>
   </widget>
   <widget class="QTableWidget" name="tableDeposit">
    <property name="geometry">
     <rect>
      <x>52</x>
      <y>328</y>
      <width>491</width>
      <height>220</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
     </font>
    </property>
    <attribute name="horizontalHeaderVisible">
     <bool>true</bool>
    </attribute>
    <attribute name="horizontalHeaderCascadingSectionResizes">
     <bool>false</bool>
    </attribute>
    <attribute name="horizontalHeaderMinimumSectionSize">
     <number>30</number>
    </attribute>
    <attribute name="horizontalHeaderDefaultSectionSize">
     <number>108</number>
    </attribute>
    <attribute name="horizontalHeaderHighlightSections">
     <bool>true</bool>
    </attribute>
    <attribute name="horizontalHeaderShowSortIndicator" stdset="0">
     <bool>false</bool>
    </attribute>
    <attribute name="horizontalHeaderStretchLastSection">
     <bool>false</bool>
    </attribute>
    <attribute name="verticalHeaderMinimumSectionSize">
     <number>40</number>
    </attribute>
    <attribute name="verticalHeaderDefaultSectionSize">
     <number>40</number>
    </attribute>
    <column>
     <property name="text">
      <string>Месяц</string>
     </property>
     <property name="font">
      <font>
       <pointsize>10</pointsize>
      </font>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Платёж</string>
     </property>
     <property name="font">
      <font>
       <pointsize>10</pointsize>
      </font>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Пополнение</string>
     </property>
     <property name="font">
      <font>
       <pointsize>10</pointsize>
      </font>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Депозит</string>
     </property>
     <property name="font">
      <font>
       <pointsize>10</pointsize>
      </font>
     </property>
    </column>
   </widget>
   <widget class="QLabel" name="label_tax_amount">
    <property name="geometry">
     <rect>
      <x>185</x>
      <y>270</y>
      <width>171</width>
      <height>16</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>15</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="text">
     <string> 0</string>
    </property>
   </widget>
   <widget class="QDoubleSpinBox" name="doubleSpinBox_add_sum">
    <property name="geometry">
     <rect>
      <x>432</x>
      <y>126</y>
      <width>111</width>
      <height>30</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="minimum">
     <double>0.010000000000000</double>
    </property>
    <property name="maximum">
     <double>99999999.989999994635582</double>
    </property>
    <property name="singleStep">
     <double>1.000000000000000</double>
    </property>
   </widget>
   <widget class="QDoubleSpinBox" name="doubleSpinBox_deposit_amount">
    <property name="geometry">
     <rect>
      <x>53</x>
      <y>67</y>
      <width>201</width>
      <height>30</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="minimum">
     <double>0.010000000000000</double>
    </property>
    <property name="maximum">
     <double>100000000.000000000000000</double>
    </property>
    <property name="singleStep">
     <double>1000.000000000000000</double>
    </property>
   </widget>
   <widget class="QDoubleSpinBox" name="doubleSpinBox_tax_rate">
    <property name="geometry">
     <rect>
      <x>183</x>
      <y>147</y>
      <width>71</width>
      <height>30</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="minimum">
     <double>-99.989999999999995</double>
    </property>
    <property name="singleStep">
     <double>0.050000000000000</double>
    </property>
   </widget>
   <widget class="QDoubleSpinBox" name="doubleSpinBox_withdrawals">
    <property name="geometry">
     <rect>
      <x>433</x>
      <y>186</y>
      <width>111</width>
      <height>30</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="minimum">
     <double>0.010000000000000</double>
    </property>
    <property name="maximum">
     <double>99999999.989999994635582</double>
    </property>
    <property name="singleStep">
     <double>1.000000000000000</double>
    </property>
   </widget>
   <widget class="QLabel" name="label_deposit_amount">
    <property name="geometry">
     <rect>
      <x>185</x>
      <y>288</y>
      <width>171</width>
      <height>16</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>15</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="text">
     <string> 0</string>
    </property>
   </widget>
   <widget class="QComboBox" name="combobox_deposit_addpay">
    <property name="geometry">
     <rect>
      <x>293</x>
      <y>126</y>
      <width>131</width>
      <height>30</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color : #FFF</string>
    </property>
    <item>
     <property name="text">
      <string>нет</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>раз в месяц</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>раз в квартал</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>раз в полгода</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>раз в год</string>
     </property>
    </item>
   </widget>
   <widget class="QLabel" name="label_16">
    <property name="geometry">
     <rect>
      <x>306</x>
      <y>109</y>
      <width>71</width>
      <height>16</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Пополнения</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_13">
    <property name="geometry">
     <rect>
      <x>53</x>
      <y>192</y>
      <width>121</width>
      <height>16</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Процентная ставка</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_20">
    <property name="geometry">
     <rect>
      <x>53</x>
      <y>287</y>
      <width>131</width>
      <height>16</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="toolTip">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Сумма на вкладе к концу срока c учетом налогов&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
    <property name="text">
     <string>Вклад к концу срока</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_11">
    <property name="geometry">
     <rect>
      <x>55</x>
      <y>49</y>
      <width>88</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Сумма вклада</string>
    </property>
   </widget>
   <widget class="QDoubleSpinBox" name="doubleSpinBox_interest_rate_d">
    <property name="geometry">
     <rect>
      <x>183</x>
      <y>186</y>
      <width>71</width>
      <height>30</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="minimum">
     <double>-99.989999999999995</double>
    </property>
    <property name="singleStep">
     <double>0.050000000000000</double>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_deposit">
    <property name="geometry">
     <rect>
      <x>381</x>
      <y>253</y>
      <width>161</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Inter</family>
      <pointsize>18</pointsize>
      <italic>false</italic>
      <bold>false</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
    border-radius: 0px;
    background-color: #FF9F06; /* Основной цвет фона */
    color: #FFF;
}

QPushButton:pressed {
    background-color: #CD8004; /* Цвет фона при нажатии */
}

QPushButton:hover {
    background-color: #FFBE56; /* Цвет фона при наведении курсора */
}</string>
    </property>
    <property name="text">
     <string>DepositCalc</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_interest_charges">
    <property name="geometry">
     <rect>
      <x>185</x>
      <y>253</y>
      <width>171</width>
      <height>16</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>15</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="text">
     <string> 0</string>
    </property>
   </widget>
   <widget class="QComboBox" name="comboBox_period_pays">
    <property name="geometry">
     <rect>
      <x>293</x>
      <y>68</y>
      <width>254</width>
      <height>30</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton { 
border-radius: 25px;
background-color : rgb(59, 60, 60);
color : #FFF
}
QPushButton:pressed {
    background-color: qlineargradient(x1: 0, y1: 0, x2: 2, y2: 2,
                                      stop: 0 #BEBEBE, stop: 1 #D7D7D7);
}

QPushButton:hover {
    background-color : rgb(79, 80, 80);
}</string>
    </property>
    <item>
     <property name="text">
      <string>ежедневный</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>раз в месяц</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>раз в год</string>
     </property>
    </item>
   </widget>
   <widget class="QLabel" name="label_12">
    <property name="geometry">
     <rect>
      <x>53</x>
      <y>113</y>
      <width>111</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Срок размещения 
(в месяцах)</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="radioButton_pay">
    <property name="geometry">
     <rect>
      <x>285</x>
      <y>20</y>
      <width>107</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Выплачивать</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_18">
    <property name="geometry">
     <rect>
      <x>53</x>
      <y>250</y>
      <width>60</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Проценты</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_14">
    <property name="geometry">
     <rect>
      <x>53</x>
      <y>154</y>
      <width>131</width>
      <height>16</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Ключевая ставка ЦБ</string>
    </property>
   </widget>
   <widget class="QComboBox" name="combobox_deposit_snyatie">
    <property name="geometry">
     <rect>
      <x>293</x>
      <y>187</y>
      <width>131</width>
      <height>30</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <item>
     <property name="text">
      <string>нет</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>раз в месяц</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>раз в квартал</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>раз в полгода</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>раз в год</string>
     </property>
    </item>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
